sql/core/src/main/scala/org/apache/spark/status/api/v1/sql/SqlResource.scala

```diff
@@ -64,6 +66,33 @@ private[v1] class SqlResource extends BaseAppResource {
     }
   }
 
+  @GET
+  @Path("streamingqueries")
```

Comment:
Shall we move the API under https://github.com/apache/spark/blob/master/streaming/src/main/scala/org/apache/spark/status/api/v1/streaming/ApiStreamingRootResource.scala?

The API can be `/applications/[app-id]/streaming/queries`

Reply:
Hmm... @HeartSaVioR WDYT?

Reply:
Having this in SQL/core module seems to be appropriate. Technically Structured Streaming is now zero related to DStream.